src/components/News/News.js

```diff
@@ -40,6 +44,8 @@ function News(props) {
         handleArticleClick={props.handleArticleClick}
         isOnMain={isOnMain}
         isLoggedIn={props.isLoggedIn}
+        savedArticles={props.savedArticles}
+        openLoginModal={props.openLoginModal}
       />
     </section>
   );
```

Comment:
In `Search.js` you need to add tag `form` because you have an input there and you need to delete `onKeyDown` because you need to add `onSubmit` to tag `form` (it handles `Enter` presses)